workflow-bot-app/src/main/java/com/symphony/bdk/workflow/api/v1/WorkflowsApi.java

```diff
@@ -126,4 +128,14 @@ ResponseEntity<List<VariableView>> listWorkflowGlobalVariables(
           + "The date is an ISO 8601 date", example = "2022-09-21T15:43:24.917Z")
       @RequestParam(required = false, name = "updated_after") Instant updatedAfter);
 
+
+  @ApiOperation("Upload a secret")
+  @ApiResponses(value = {@ApiResponse(code = 204, message = "")})
+  @PostMapping("/secrets")
+  ResponseEntity<Void> uploadSecret(@RequestBody SecretView secrete);
+
+  @ApiOperation("Delete a secret")
+  @ApiResponses(value = {@ApiResponse(code = 204, message = "")})
+  @DeleteMapping("/secrets/{key}")
+  ResponseEntity<Void> deleteSecret(@PathVariable("key") String secretKey);
```

Comment:
Probably need a list call as well, else it's not clear how the `key` could be discovered for deletion